ng2-components/ng2-activiti-form/src/components/activiti-form.component.ts

```diff
@@ -41,6 +41,9 @@ export class ActivitiForm implements OnInit, AfterViewChecked, OnChanges {
     static CUSTOM_OUTCOME_ID: string = '$custom';
 
     @Input()
+    form: FormModel;
```

Comment:
It is in the readme in the `Advanced Properties` section since the beginning, it was just missing the `@Input` decorator.